pipelines/jobs/configurations/jdk11u.groovy

```diff
@@ -51,6 +51,9 @@ targetConfigurations = [
         ],
         "aarch64LinuxXL": [
                 "openj9"
+        ],
```

Comment:
Just to confirm, we DO want to have riscv run in the nightlies yes? Otherwise, we shouldn't need this

Reply:
It's quick enough and reliable enough (based on https://ci.adoptopenjdk.net/job/jdk11-linux-riscv-openj9-build/) that I'm happy with it being run regularly now